commit 6da24020a4ab6b69e51ef8419888ad2b68395733
Author: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
Date:   2019-07-05T08:47:32-06:00 (3 months ago)
    
    MANTA-4390 Unwrap rpc data array in node-boray's deleteObjectNoVnode function

diff --git a/lib/objects.js b/lib/objects.js
index d7372f3..0dcd7c5 100644
--- a/lib/objects.js
+++ b/lib/objects.js
@@ -391,7 +391,12 @@ function deleteObjectNoVnode(rpcctx, owner, bucket_id, name, req_id, callback) {
             return;
         }
 
-        callback(null, data);
+        if (data.length === 1) {
+            callback(null, data[0]);
+        } else {
+            callback(new VError('expected 1 data messages, found %d',
+                data.length));
+        }
     });
 }
 
